src/components/svg-canvas-graph.tsx

```diff
@@ -285,6 +335,55 @@ const SvgCanvas = () => {
                     styleAttrs={{ color: theme }}
                 />
             )}
+            {mode === 'select' && selectStart.x != 0 && selectStart.y != 0 && (
```

Comment:
This could be moved into `svg-wrapper.tsx` if we do not support select from nodes. Also, no more `selectStart` and `selectMoving` need to be shared between this and `svg-wrapper.tsx`.

Reply:
But when the mouse go above a node, it will be recorded by handleMoving of canvas rather than it of wrappers.

Reply:
I have moved it to `svg-wrapper.tsx` .